target/mips/translate_init.c.inc

```diff
@@ -493,7 +493,7 @@ const mips_def_t mips_defs[] =
 #if defined(TARGET_MIPS64)
 // 8 byte counter for CPU revision. Bump this every time there is major incompatible change
 // so that we can print a warning when booting CheriBSD with a too old QEMU
-#define CHERI_PROCESSOR_REVISION_COUNTER 0x5
+#define CHERI_PROCESSOR_REVISION_COUNTER 0x6
```

Comment:
CheriBSD definitely doesn't fall over if the version is too new?

Reply:
It only checks for a minimum version not a maximum: https://github.com/CTSRD-CHERI/cheribsd/blob/18a15b8ae03424c5acd265e31f51eb7882a05a80/sys/mips/mips/cpu.c#L362

Reply:
Ah good, was struggling to find that code